src/styles/index.css

```diff
@@ -0,0 +1,103 @@
+@import url('https://fonts.googleapis.com/css2?family=Mukta+Vaani:wght@400;500;600;800&display=swap');
+
+html {
+  font-size: 100%;
+  box-sizing: border-box;
+}
```

Comment:
- [ ] Great working on some of the changes required! Please kindly ensure to do the following:

1. The user icon is missing from the UI, but I observed that it was there before the previous review. ℹ️ 
2. Kindy style the `BOOKSTORECMS` text according to the given template. I really appreciate your creativity actually, but let's follow the project requirement. ℹ️ **Note: I noticed it was re-styled after the previous review.**
3. Kindly align the form element to be inline with other elements on the page in accordance to the given template. 👍
4. It will be great if the vertical scroll bar can be removed from the UI, this will enhance the user interaction. 👍 **This was not there in the previous review.**

## Your design
<img width="811" alt="Untitled" src="https://github.com/Oscah01/the-book-store/assets/103386183/4fab143e-bdc0-479f-8e12-34a112a5cb0b">

Reply:
@OyePriscilla I think it's not fair what you did

Reply:
@Oscah01 I understand your plight, but all the same I think what I did was still within the scope of the last review.